README.md

```diff
@@ -139,10 +139,6 @@ Example command:
 
```

Comment:
## 1.
![image](https://user-images.githubusercontent.com/93039359/219118428-7f02b93f-34d3-469f-adc5-558da9fe70c3.png)
## 2.
![image](https://user-images.githubusercontent.com/93039359/219118623-52c3207b-7028-41ee-b692-b0c36f215494.png)
## 3.
![image](https://user-images.githubusercontent.com/93039359/219118696-529a291f-8383-4849-895d-b61863eba8a4.png)

**[Optional]**

- [ ] Kindly remove these **section descriptions** and **NOTE** because these are just instructions for you :+1: Kindly delete them to make your readme file clean and descriptive :+1: